internal/worker/bootstrap/service.go

```diff
@@ -98,6 +98,15 @@ type UserService interface {
 	// the user will be added with an activation key.
 	AddUser(ctx context.Context, arg userservice.AddUserArg) (user.UUID, []byte, error)
 
+	// AddExternalUser adds a new external user to the database and does not set a
+	// password or activation key.
+	// The following error types are possible from this function:
+	//   - accesserrors.UserNameNotValid: When the username supplied is not valid.
+	//   - accesserrors.UserAlreadyExists: If a user with the supplied name already exists.
+	//   - accesserrors.CreatorUUIDNotFound: If a creator has been supplied for the user
```

Comment:
question: Under what conditions can a creator not be specified?

Reply:
Good point, I copied this from a method where it was optional. I've changed the wording